.eslintignore

```diff
@@ -9,4 +9,5 @@ packages/react-native-editor/bundle
 playground/dist
 vendor
 wordpress
+*.d.ts
```

Comment:
It's fine if you are adding `*.d.ts` in Gutenberg, but it won't for other projects that use `@wordpress/eslint-plugin` or `@wordpress/scripts`. Now the question is how we can remove `*.d.ts` file from listing directly in ESLint plugin, any ideas?

Reply:
Maybe they can be linted and this was overzealous. The issue I experienced seems to be this which has apparently been addressed: https://github.com/typescript-eslint/typescript-eslint/issues/2456 😕